examples/win-offsets.c

```diff
@@ -32,12 +32,21 @@
 #include <stdio.h>
 #include <inttypes.h>
 #include <glib.h>
+#include <signal.h>
+
+vmi_instance_t vmi;
+
+void sigint_handler()
+{
+    vmi_resume_vm(vmi);
+    vmi_destroy(vmi);
+    exit(1);
```

Comment:
You would leak the `config` ghashtable if you exit here. It's not like it would matter for this particular application but this sets a bad example. Just make sure that ghashtable gets freed, make the `done:` part into a standalone function and call it in both main and the sigint handler.

Reply:
:+1: